baselines/FedStar/README.md

```diff
@@ -0,0 +1,90 @@
+
+****Contributors:**** : Raj Parekh.
+
+
+## Experimental Setup
+
+****Task:**** :warning: *_what’s the primary task that is being federated? (e.g. image classification, next-word prediction). If you have experiments for several, please list them_*
+
+****Model:**** :warning: *_provide details about the model you used in your experiments (if more than use a list). If your model is small, describing it as a table would be :100:. Some FL methods do not use an off-the-shelve model (e.g. ResNet18) instead they create your own. If this is your case, please provide a summary here and give pointers to where in the paper (e.g. Appendix B.4) is detailed._*
+
+****Dataset:**** :warning: *_Earlier you listed already the datasets that your baseline uses. Now you should include a breakdown of the details about each of them. Please include information about: how the dataset is partitioned (e.g. LDA with alpha 0.1 as default and all clients have the same number of training examples; or each client gets assigned a different number of samples following a power-law distribution with each client only instances of 2 classes)? if  your dataset is naturally partitioned just state “naturally partitioned”; how many partitions there are (i.e. how many clients)? Please include this an all information relevant about the dataset and its partitioning into a table._*
+
+****Training Hyperparameters:**** :warning: *_Include a table with all the main hyperparameters in your baseline. Please show them with their default value._*
+
+
+## Environment Setup
+
+:warning: _The Python environment for all baselines should follow these guidelines in the `EXTENDED_README`. Specify the steps to create and activate your environment. If there are any external system-wide requirements, please include instructions for them too. These instructions should be comprehensive enough so anyone can run them (if non standard, describe them step-by-step)._
+
+
+## Running the Experiments
+
+:warning: _Provide instructions on the steps to follow to run all the experiments._
+```bash  
+tar -xvf data_splits.tar
+
+pip install tensorflow
+
+pip install flwr
+
+mkdir datasets && cd datasets
+
+#for setting up speech commands data set
+mkdir speech_commands && cd speech_commands
+wget http://download.tensorflow.org/data/speech_commands_v0.02.tar.gz -O train.tar.gz && mkdir -p ./Data/Train && tar -zxvf ./train.tar.gz  -C ./Data/Train
+wget http://download.tensorflow.org/data/speech_commands_test_set_v0.02.tar.gz -O test.tar.gz && mkdir -p ./Data/Test && tar -zxvf ./test.tar.gz  -C ./Data/Test
+mv ./Data/Train/_background_noise_ ./Data/Train/_silence_
```

Comment:
If possible, put this in a bash script so it's easier for people to run.

Reply:
Added setup_project.sh for the initial setup.